internal/db/migrations/2024-10-14.0-alter_tss_transaction_tries.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+
+ALTER TABLE tss_transaction_submission_tries
+    DROP column status,
+    ADD column status TEXT NOT NULL,
+    ADD COLUMN code INTEGER NOT NULL,
+    ADD COLUMN result_xdr TEXT NOT NULL;
```

Comment:
Why not rename the current column `status` to `code` and then add a new column called `status`? This way we don't lose data by dropping a column.

```suggestion
    RENAME COLUMN status TO code,
    ADD column status TEXT NOT NULL,
    ADD COLUMN result_xdr TEXT NOT NULL;
```

Reply:
Changed this. Although I had to put the RENAME COLUMN clause in it's own ALTER TABLE for tests to pass. Apparently its syntactically incorrect to have all those changes in the same ALTER TABLE statement